cognite_toolkit/examples/cdf_oid_example_data/raw/timeseries2assets.yaml

```diff
@@ -1,2 +1,2 @@
-dbName: {{raw_db}}
+dbName: {{raw_db_timeseries}}
```

Comment:
Why do you want to split up all the tables in individual databases? You can do access scoping on table level or do I remember wrongly?

Reply:
since we never collect tables from multiple source systems into one raw db - again thought we where trying to set up something that looked like a proper CDF project - not a quick fix demo....